pkgs/applications/networking/instant-messengers/blink/default.nix

```diff
@@ -1,68 +1,93 @@
-{ lib, fetchdarcs, python2Packages, libvncserver, zlib
-, gnutls, libvpx, makeDesktopItem, mkDerivationWith }:
-
-mkDerivationWith python2Packages.buildPythonApplication rec {
+{ lib, python37Packages, libvncserver, xorg, makeDesktopItem
+, mkDerivationWith, fetchFromGitHub, callPackage }:
+let
+  # using Python 3.7 because python3-application requires older than 3.8
+  python37Packages' = python37Packages.override {
```

Comment:
should probably use packageOverrides: https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#overriding-python-packages-overriding-python-packages

Reply:
Good to know, thanks.
I ended up bringing in some new commits in deps, and no longer have to use Python 3.7, so I was able to remove this override.